README.md

```diff
@@ -438,22 +446,78 @@ _To be added_
 
 ### 3. Other configuration options
 
-SGX-LKL-OE has a number of other configuration options e.g. for configuring the
-in-enclave scheduling, network configuration, or debugging/tracing. To see all
-options, run:
+SGX-LKL-OE has a number of other configuration options, e.g. for configuring the
+disks, in-enclave scheduling, network configuration, or debugging/tracing.
+
+There are two ways to specify configuration options: environment variables and JSON files.
+
+During debugging, it can be convenient to specify debugging options as environment
+variables. Note that for debugging options to have an effect, SGX-LKL must be built
+with `DEBUG=true`.
+To see a list of all environment configuration variables, run:
 ```
-sgx-lkl-run-oe --help
+sgx-lkl-run-oe --help-config
 ```
 
-Note that for the debugging options to have an effect, SGX-LKL must be built
-with `DEBUG=true`.
+The recommended way to configure SGX-LKL is through host config and app config JSON files.
+Using JSON files is also required when creating Confidential Containers (see next section).
+To create initial host and app config files in the current directory, run:
+```
+sgx-lkl-cfg create --disk sgxlkl-disk.img
+```
+Note that you may have to adapt the generated files to your needs.
+See the `tests/containers/cc` folder for example host and app config files.
+You can then run SGX-LKL using the host and app config files alone:
+```
+sgx-lkl-run-oe --hw-debug --host-config=host-config.json --app-config=app-config.json
+```
+Note that environment variables can still be used to specify additional options.
+This is especially useful for host configuration options that are only known
+at runtime in the target system where SGX-LKL will run, for example the name
+of the network tap device (`SGXLKL_TAP`).
```

Comment:
Do we want to keep environment variables that refer to the host_config in a RELEASE build?

Reply:
Yeah, I think this may be useful for infrastructure injecting information (like the tap device to use).

Reply:
And since there is no difference in security for the host config I wouldn't make the behaviour difference between build types, simply for consistency.